### DISCUSSION TITLE
Any planned support for multiple device tests

### DISCUSSION THREAD

**Discussion by yschimke** [mobile-dev-inc/Maestro#177] [Ideas]
Specifically for Wear, it would be great to test component app functionality on the Watch, paired with the mobile app.

My needs are specifically tests that interact with multiple devices to complete a single test.

**Comment by readbeard** [comment]
It could be interesting also if there's a plan to support sharding!

**Comment by axelniklasson** [comment]
Hey folks! This is not something we are actively planning to tackle soon, but please file a feature request [here](https://github.com/mobile-dev-inc/maestro/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.md&title=%5BFeature+Request%5D) if this is something you are still wanting to see added to Maestro. Thanks!
